test/hummingbot/connector/exchange/bybit/test_bybit_auth.py

```diff
@@ -103,10 +95,9 @@ def _generate_signature(self, params: Dict[str, Any]) -> str:
     def _params_expected(self, request_params: Optional[Mapping[str, str]]) -> Dict:
         request_params = request_params if request_params else {}
         params = {
-            'timestamp': 1000000,
             'api_key': self.api_key,
         }
         params.update(request_params)
         params = OrderedDict(sorted(params.items(), key=lambda t: t[0]))
-        params['sign'] = self._generate_signature(params=params)
+        # params['sign'] = self._generate_signature(params=params)
```

Comment:
why this is commented? is not used?

Reply:
No needed there, though I have just pushed a commit that adds rest/ws signature tests.